CMakeLists.txt

```diff
@@ -81,11 +81,8 @@ tristate_feature_auto(ENABLE_RIPEMD160 "Enable RIPEMD-160 hash support.")
 
 option(ENABLE_CRYPTO_REFRESH "Enable crypto-refresh support (v6)")
 option(ENABLE_PQC "Enable PQC support - requires ENABLE_CRYPTO_REFRESH")
-
-if((NOT ENABLE_CRYPTO_REFRESH) AND ENABLE_PQC)
-  message(FATAL_ERROR "ENABLE_PQC requires ENABLE_CRYPTO_REFRESH")
-endif()
-
+option(ENABLE_PQC_MLKEM_IPD "Use ML-KEM-ipd instead of Kyber")
```

Comment:
Do you have some special need to have these two options in CMake file and config.h.in, which would be available for every user and could confuse them/library maintainers? It could be made without declaring an option in a way like:

> cmake -DENABLE_PQC_MLKEM_IPD=On ...

```
if (DEFINED ENABLE_PQC_MLKEM_IPD)
  add_definitions(-DENABLE_PQC_MLKEM_IPD)
endif()
```